src/components/combo_box/combo_box.tsx

```diff
@@ -142,6 +142,16 @@ export interface _EuiComboBoxProps<T>
    * Specifies that the input should have focus when the component loads
    */
   autoFocus?: boolean;
+  /**
+   * Creates an `aria-label` attribute when using the Combobox without a visible label from `<EuiFormRow />`
```

Comment:
Minor sentence casing suggestion

```suggestion
   * Creates an `aria-label` attribute when using the combo box without a visible label from `<EuiFormRow />`
```

Really love the detail and context in these prop docs! Awesome work!